pkg/commands/server/pluginshealth.go

```diff
@@ -44,11 +51,21 @@ var pluginsHealthCommand = cli.Command{
 // cmPluginsHealth is the action for the pluginsHealthCommand. It makes a "plugins" request
 // against the active Synse Server instance and returns plugins' health information.
 func cmdPluginsHealth(c *cli.Context) error {
-	plugins, err := client.Client.Plugins()
+	err := utils.RequiresArgsInRange(0, 1, c)
+	if err != nil {
+		return err
+	}
+
+	plugins, err := getPlugins(c.Args().Get(0), c)
 	if err != nil {
 		return err
 	}
 
+	// FIXME: Should we return nil here? Refer to #179.
```

Comment:
good question. 

my initial thought is that if synse server has no plugins configured, then no plugins should be displayed. If returning nil does that, then I think this is fine. I think that the command should still return empty responses, e.g.

yaml:
```yaml

```

json:
```json
[]
```

pretty:
```
TAG	PROTOCOL	 ADDRESS	 VERSION	 STATUS
```

that way, if someone is using the CLI programmatically, they get back a valid empty response (instead of just nothing)

Reply:
I've been playing around with this and I think I know why some commands return nothing, instead of empty responses. Consider a `synse server read` example.

Issusing a `pretty` read for "foo" gets no results as expected:
```
➜  synse-cli git:(plugins-filter) ✗ ./build/synse --format pretty server read foo
INFO      TYPE      VALUE     UNIT      TIMESTAMP
```
However, changing the format to `yaml`, I get:
```
➜  synse-cli git:(plugins-filter) ✗ ./build/synse --format yaml server read foo
no data to write
```

Similar thing happens with `synse server plugins health` above. I didn't realize that I skipped this error until now. Also, this problem is still living in the master branch and there is currently a FIXME flag around that. https://github.com/vapor-ware/synse-cli/blob/691de75075ab34f3af34bca8a57f4c3819f3cb34/pkg/commands/server/read.go#L115

For this PR, I will just keep it as it is and revisit in the future.

Reply:
yeah, that makes total sense. the formatter was thrown together a little quickly, so i'm sure it can be improved in a lot of ways. thanks for looking into this!